backend/day-05/day-5-challenge.py

```diff
@@ -0,0 +1,44 @@
+from difflib import get_close_matches
+import random
+
+options = {
+    "scissor": {
+        "wins to": "paper",
+        "losses to": "rock",
+    },
+    "rock": {
+        "wins to": "scissor",
+        "losses to": "paper"
+    },
+    "paper": {
+        "wins to": "rock",
+        "losses to": "scissor"
+    }
+}
+
+def validate_input(player_input):
+    valid_choices = options.keys()
+    closest_match = get_close_matches(player_input, valid_choices, n=1, cutoff=0.6)
+
+    if closest_match:
+        return closest_match[0]
+    else:
+        print("INVALID INPUT")
+        exit()
+
+def evaluate(player, computer):
+    if options[player]["wins to"] == computer:
+        print("You win!")
+    elif options[player]["losses to"] == computer:
+        print("Computer wins!")
+    elif player == computer:
```

Comment:
Since you have a finite set of conditions, i believe you can just use set this as else instead of elif. 
It doesn't really make of a difference though haha

Reply:
talaga